test/module/irohad/ametsuchi/ametsuchi_mocks.hpp

```diff
@@ -155,8 +158,8 @@ namespace iroha {
       MOCK_METHOD2(
           apply,
           bool(const model::Block &,
-               std::function<bool(
-                   const model::Block &, WsvQuery &, const hash256_t &)>));
+               std::function<
+                   bool(const model::Block &, WsvQuery &, const hash256_t &)>));
```

Comment:
Broken clang format. Pls, recheck all files in PR and fix it.

Reply:
That's how it formatted this file